discovery-infra/bootstrap_in_place.py

```diff
@@ -0,0 +1,171 @@
+import os
+import shutil
+import shlex
+import logging
+import yaml
+
+import waiting
+
+from oc_utils import get_operators_status
+from download_logs import download_must_gather
+from test_infra import utils, consts
+from test_infra.tools.assets import NetworkAssets
+from test_infra.controllers.node_controllers.ssh import SshConnection
+from test_infra.controllers.node_controllers.terraform_controller import TerraformController
+
+BUILD_DIR = "build"
+INSTALL_CONFIG_FILE_NAME = "install-config.yaml"
+IBIP_DIR = os.path.join(BUILD_DIR, "ibip")
+RESOURCES_DIR = os.path.join("discovery-infra", "resources")
+INSTALL_CONFIG = os.path.join(IBIP_DIR, INSTALL_CONFIG_FILE_NAME)
+INSTALLER_BINARY = os.path.join(BUILD_DIR, "openshift-install")
+EMBED_IMAGE_NAME = "installer-SNO-image.iso"
+KUBE_CONFIG = os.path.join(IBIP_DIR, "auth", "kubeconfig")
+MUST_GATHER_DIR = os.path.join(IBIP_DIR, "must-gather")
+SOSREPORT_SCRIPT = os.path.join(RESOURCES_DIR, "man_sosreport.sh")
+SSH_KEY = os.path.join("ssh_key", "key")
+
+
+def installer_generate():
+    logging.info("Installer generate manifests")
+    utils.run_command(f"{INSTALLER_BINARY} create manifests --dir={IBIP_DIR}")
+    logging.info("Installer generate ignitions")
+    # TODO delete
+    shutil.copy(f"{RESOURCES_DIR}/sno_manifest.yaml", os.path.join(IBIP_DIR, "openshift"))
```

Comment:
@eranco74 still needed?

Reply:
I'll remove it once we move to quay.io/eranco74/ocp-release:bootstrap-in-place